source/track_thin.f90

```diff
@@ -456,14 +456,7 @@ subroutine trauthin(nthinerr)
       write(lout,"(a)") ""
       write(lout,"(a)") "TRACKING> Calling thin6d subroutine"
       write(lout,"(a)") ""
-      if (do_coll) then
-        call collimate_init()
-        call collimate_start_sample(1) ! Changed to only do 1 sample
-      endif
       call thin6d(nthinerr)
-      if (do_coll) then
-        call collimate_end_sample(1) ! Changed to only do 1 sample
-      endif
```

Comment:
I would have left these calls here, actually... I think it's cleaner

Reply:
As we discussed, the reason this was moved to main_cr is so it's called before pre-track. This is necessary because the collimation module generates beam distribution, which must be before the initial_state file is written.